Bottom: 894a26efd117c5d984796c47fe8e4c4ed3b41872
Top:    d5e6b50fedd06021794610b39393d86749aa9676
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-01 16:14:10 +0800

Refresh of tuntap-switch-to-use

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 0a31347..2f75e86 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -113,7 +113,6 @@ do {								\
 } while (0)
 #endif
 
-#define TUN_HEADROOM 256
 #define TUN_RX_PAD (NET_IP_ALIGN + NET_SKB_PAD)
 
 /* TUN device flags */
@@ -1634,7 +1633,7 @@ static struct sk_buff *tun_build_skb(struct tun_struct *tun,
 	rcu_read_lock();
 	xdp_prog = rcu_dereference(tun->xdp_prog);
 	if (xdp_prog)
-		pad += TUN_HEADROOM;
+		pad += XDP_PACKET_HEADROOM;
 	buflen += SKB_DATA_ALIGN(len + pad);
 	rcu_read_unlock();
